src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/missingswitchdefault/InputMissingSwitchDefaultCheckSwitchExpressionsFive.java

```diff
@@ -0,0 +1,53 @@
+/*
+MissingSwitchDefault
+
+
+*/
+
+//non-compiled with javac: Compilable with Java14
+package com.puppycrawl.tools.checkstyle.checks.coding.missingswitchdefault;
+
+public class InputMissingSwitchDefaultCheckSwitchExpressionsFive{
```

Comment:
If we can't meaningfully name test classes/files, it is a good indication that we need to have more focus on the input.

Reply:
Renamed @nrmancuso

Reply:
Renamed, but now we have another with strange numerical naming scheme.